src/Saunter/Saunter.csproj

```diff
@@ -39,6 +38,8 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
+    <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.2.0" />
```

Comment:
The reference can actually be removed entirely. It's included in `Microsoft.AspNetCore.App`.

Reply:
fixed